content/ja/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -0,0 +1,46 @@
+---
+title: CronJob
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+_CronJob_ は時刻ベースのスケジュールによる[Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/)を作成します。
+
+_CronJob_ オブジェクトとは _crontab_ (cron table)ファイルでみられる一行のようなものです。指定のスケジュールの基づき、定期的にジョブが実行されます。
+スケジュールでは[Cron](https://en.wikipedia.org/wiki/Cron)形式で記述されます。
```

Comment:
Maybe link to https://ja.wikipedia.org/wiki/Cron instead? (redirects to _crontab_; the important difference is that it's in Japanese).

Reply:
こちら直訳で対応します。
上記の日本語リンクはご対応する感じでよろしいでしょうか？

Reply:
https://ja.wikipedia.org/wiki/Cron doesn't exist. I wonder it is more helpful to link there and reference Wikipedia's page in Japanese that is about cron, rather than to a page that is in English.

I am assuming that a Japanese speaker who prefers English will look at the English documentation, and that anyone preferring Kubernetes documentation in Japanese will also prefer Wikipedia in Japanese.

Reply:
Thank you for your advise. I understood it and refflect it soon.

Reply:
日本語で対応する感じで良いと思います.